### DISCUSSION TITLE
Mouse pointer capture is lost when moving out of Win2D Canvas control

### DISCUSSION THREAD

**Discussion by d2phap** [microsoft/WindowsAppSDK#1908] [Q&A]
I'm using Win2D to draw an image, and implement panning feature. 
The problem is when I press, and move the mouse cursor out of the Canvas control, the pointer capture is lost.

Any idea to prevent this?
Something similar to the `setPointerCapture` API in JavaScript: https://developer.mozilla.org/en-US/docs/Web/API/Element/setPointerCapture

Thanks

```xaml
<win2d:CanvasControl
  x:Name="canvas"
  Draw="canvas_Draw"
  PointerEntered="canvas_PointerEntered"
  PointerMoved="canvas_PointerMoved"
  PointerPressed="canvas_PointerPressed"
  PointerReleased="canvas_PointerReleased"
  PointerWheelChanged="canvas_PointerWheelChanged" />
```

![image](https://user-images.githubusercontent.com/3154213/145983894-ec9f9a08-5e3a-40cd-982d-9f602af27aea.png)

**Comment by ghost** [comment]
@andrewleader
